### PR TITLE
Disable pointer events on disabled elements within a tooltip container

### LINKED ISSUE #755: Tooltips on disabled table checkboxes don't dismiss correctly
If you quickly move out of the table row the tooltip doesn't dismiss.

## Code used
```
<td>
    <div data-toggle="tooltips" title="first tooltip" data-placement="right">
        <input type="checkbox" disabled />
    </div>
</td>
```

tooltips

Related to https://jadultd.atlassian.net/browse/XFP-2193

### PR DESCRIPTION
**Pull request by Stanton**
Fixes #755 issue where tooltips on disabled checkboxes within a table wouldn't dismiss correctly if the cursor was moved too quickly away from the element.

**Issue**
Screenshot

Solution was to disable the pointer events on disabled elements within a tooltip wrapper so they don't interfere with the `mouseenter` and `mouseleave` events. Tooltip documentation has been updated to detail this behaviour.

### CODE REVIEW

File: stylesheets/_component.tooltips.scss [hunk 1]
```diff
@@ -60,74 +60,80 @@
...
 }
 
 div[data-toggle="tooltips"] {
-    display: inline-block;
+  display: inline-block;
+}
+
+// Disable pointer events on disabled elements within a tooltip to prevent the issue where
+// moving the cursor fast across a disabled element causes the mouseleave event not to fire
+[data-toggle="tooltips"] > [disabled] {
```
Comment (Stanton) [inline stylesheets/_component.tooltips.scss]: Actual change is here, rest of diff is documentation and tab normalisation

File: docs/_sass/_base.scss [hunk 2]
```diff
@@ -70,3 +70,37 @@ table:not(.table) {
...
+    font-size: .8rem;
+    margin-top: 1rem;
+  }
+}
+
+.callout {
+  border-bottom-right-radius: 50px;
+  border-top-right-radius: 50px;
+  margin-left: -50px;
+  padding: 1rem 0 1rem 50px;
```
Comment (nvasileiadis) [inline docs/_sass/_base.scss]: If you started using rem in padding why not in margin too?

Comment (Stanton) [reply docs/_sass/_base.scss]: Fixed left & right margins, relative vertical rhythm.

**Comment by Stanton** [comment]
# Breaking change quick checklist

The following points should be considered as an early-warning of introducing a breaking change to a Continuum platform product. This is not an exhaustive list of what constitutes a breaking change.

| Does this pull request... | Yes / No |
| ---- | ---- |
| require changes to `main.js` | ✅ No  |
| require changes to `index.js` | ✅ No |
| require changes to `pulsar.scss` | ✅ No |
| require changes to `gruntfile.js` | ✅ No |
| require changes to `package.json` (not including dev dependencies) | ✅ No |
| require changes to `base.html.twig` (or other core views like header/footer) | ✅ No |
| require new arguments to be passed to a js component | ✅ No |
| require markup changes to maintain functionality | ✅ No |

**Pull request merged**